rmk/src/channel.rs

```diff
@@ -15,6 +15,9 @@ use {crate::split::SplitMessage, embassy_sync::pubsub::PubSubChannel};
 
 pub const EVENT_CHANNEL_SIZE: usize = 16;
 pub const REPORT_CHANNEL_SIZE: usize = 16;
+pub const CONTROLLER_CHANNEL_SIZE: usize = 16;
```

Comment:
Could you make these constants configurable? Just like https://github.com/HaoboGu/rmk/pull/351 does